### ISSUE TITLE
Commands that abort in interactive mode abort the whole session

### ISSUE THREAD

**Issue by andy-maier** [pywbem/pywbemtools#394]
Example: 
```
$ pywbemcli
pywbemcli> -m m -s s connection save c
Error: --mock-server: File: "/Users/maiera/PycharmProjects/pywbem/pywbemtools/m" extension: "" not valid. "py" or "mof" required
Aborted!
$
```
I think it should only abort the invalid command, but keep the interactive session.

That is the case for example for invalid options and invalid commands:
```
$ pywbemcli
pywbemcli> x
Usage: pywbemcli  [GENERAL-OPTIONS] COMMAND [ARGS]...
Try "pywbemcli  -h" for help.

Error: No such command "x".
pywbemcli>
```

**Comment by andy-maier** [comment]
We discussed this and the only situation to abort the session is when the repo was modified and something fails (e.g. MOF compiler error, or Python script unhandled exception), because that could mean the repo was corrupted.

**Comment by andy-maier** [comment]
I have grabbed this one and started working on it.

**Comment by andy-maier** [comment]
Karl had already started on this one and this issue is addressed by his PR #405.
The changes I had started will be based on that PR.
